IMPEDANCE_STIFFNESS_LIST:
  - 1000
  - 1000
  - 1000
  - 100
  - 30 
  - 100

TORQUE_UPPER: # default [20.0, 20.0, 18.0, 18.0, 16.0, 14.0, 12.0]
  - 40.0
  - 40.0
  - 36.0
  - 36.0
  - 32.0
  - 28.0
  - 24.0

FORCE_UPPER: # default [20.0, 20.0, 20.0, 25.0, 25.0, 25.0]
  - 100.0
  - 100.0
  - 100.0
  - 25.0
  - 25.0
  - 25.0

NMAX: 40  # N (maximum normal force robot can apply)
RATE: 30  # hz (control rate)

# concavity * (dw^T * v - k_XI * XI_err)^2
K_THETA: 300.      
CONCAVITY_THETA: 60
K_S: 1.
CONCAVITY_S: 0.3
CONCAVITY_X: 60
K_X: 108.

S_SCALE: 0.0005
X_SCALE: 0.006

REGULARIZATION_CONST: 0.00001
TORQUE_MARGIN_FACTOR: 0.002

INTEGRAL_MULTIPLIER: 5.


TRUST_REGION_LIST: 
  - 1.
  - 1.
  - 3.
  - 3.
  - 1.
  - 1. 
  - 1. 
  - 1.


